packages/mobile/src/account/saga.ts

```diff
@@ -97,6 +103,21 @@ function* migrateAccountToProperBip39() {
   }
 }
 
+function* clearStoredAccountSaga({ account }: ClearStoredAccountAction) {
+  try {
+    yield call(removeAccountLocally, account)
+    yield call(clearStoredMnemonic)
+    yield call(ValoraAnalytics.reset)
+    yield call(firebaseSignOut, firebase.app())
+    // TODO: Remove the account from geth.
```

Comment:
Until we do this, what happens if a user removes the account then restores it using their backup key?

Reply:
I addressed the rest of the comments, but couldn't fix this one yet. Without removing the account from geth one cannot restore the same account in the same phone, it throws an error. To remove the account, I think we need to add a endpoint in the RPC listener on the blockchain that allows deleting the account.